architecture/core_concepts/routes.adoc

```diff
@@ -1052,3 +1053,53 @@ spec:
     in the subdomain `example.com`. `*.example.com` is the subdomain for host
     name `wildcard.example.com` to reach the exposed service.
 ====
+
+[[disable-namespace-ownership-check]]
+== Disabling Namespace Ownership Check
```

Comment:
I think we probably need a big disclaimer/warning/something up here that says "hey, this feature is not really recommended as it can allow any namespace to be granted claims to hosts that overlap (either wildcard subdomains or paths for a regular host) and strong wording like its not recommended doing this unless you really want to and your cluster policy has locked down route creation by end users/a more stringent policy in place`. We did this specifically for a hosting provider but its a knob that has probably lesser use for/by others.